test/multihash.spec.js

```diff
@@ -0,0 +1,78 @@
+import { Piece } from '@web3-storage/data-segment'
+import { deriveBuffer } from './util.js'
+import { parse as parseLink } from 'multiformats/link'
+import * as Hasher from '../src/multihash.js'
+
+/**
+ * Module is generated from `./commp.csv` using prepare script.
+ * @see https://github.com/hugomrdias/playwright-test/issues/544
+ */
+import vector from './commp/vector.js'
+
+/**
+ * @type {import("entail").Suite}
+ */
+export const testMultihash = {
+  ...Object.fromEntries(
+    Object.values(vector).map((data) => [
+      `${data.in.contentSize}\t\t${data.in.cid}`,
+      async (assert) => {
+        const payload = deriveBuffer(data.in.contentSize)
+        const root = parseLink(data.out.cid).multihash.digest
+        const height = Piece.PaddedSize.toHeight(BigInt(data.out.paddedSize))
+
+        {
```

Comment:
Why are using brackets here?

Reply:
Just creating scopes so I don't have to come up with three different names for `digest`. It also kind of provides some separation, but if this is confusing I can remove them and call `digest1`, `digest2`.

I might actually generate separate test functions instead.